src/fluent-plugin-mdsd/lib/fluent/plugin/out_mdsd.rb

```diff
@@ -28,6 +28,8 @@ def initialize()
         config_param :conn_retry_timeout_ms, :integer, :default => 60000
         desc 'the field name for the event emit time stamp'
         config_param :emit_timestamp_name, :string, :default => "FluentdIngestTimestamp"
+        desc 'the maximum record size to emit'
```

Comment:
Ok. I added it as a constant MDSD_MAX_RECORD_SIZE and then use the min of that and the configured param. If you would like i could just use the constant and remove it as a configured parameter. I just thought it would be a good idea to have it as param in case it changes in mdsd then the user could update it without a new version of the gem.